snuba/datasets/discover.py

```diff
@@ -211,6 +213,7 @@ def __init__(self) -> None:
                 ("span_id", Nullable(UInt(64))),
                 ("transaction_hash", Nullable(UInt(64))),
                 ("transaction_op", Nullable(String())),
+                ("transaction_status", WithDefault(UInt(8), UNKNOWN_SPAN_STATUS)),
```

Comment:
I think this should be Nullable in discover:
Integer for transactions and NULL for errors. 
Is it what you are planning ?

Reply:
Yes, You're right.